client/src/outside/components/EhrSeedDataList.vue

```diff
@@ -28,32 +28,54 @@
 
               // td {{sv._id}}
               td(v-if="isDevelopingContent && !sv.isDefault",class="seed-actions")
+               ui-button(v-on:buttonClicked="duplicateSeed(sv)", secondary, :title="`Copy of ${sv.name}`") Duplicate
                ui-button(v-on:buttonClicked="downloadSeed(sv)", , v-bind:secondary="true", class="dwn") Download
                ui-button(v-on:buttonClicked="showEditDialog(sv)", v-bind:secondary="true") Edit description
                ui-button(v-on:buttonClicked="gotoEhrWithSeed(sv)") View/edit seed
       ui-agree(ref="aggreeDialog")
     ehr-seed-data-dialog(ref="theDialog", @showDialog="showDialog")
+    ui-confirm(
+      ref="confirmDialog",
+      @confirm="confirmSeedDuplication",
+      @abort="cancelSeedDuplication",
+      @cancel="cancelSeedDuplication",
+      set-footer
+      )
+      div(slot="extra", style="margin-top: calc(40vh - 25vh);") 
+        input(
+          class="checkbox",
+          type="checkbox",
+          v-model="duplicateDocuments"
+        )
+        span Add associated documents?
```

Comment:
Apologies for not seeing this sooner. We do not need this extra section. It is meant for the future when each seed has documents. For now, seeds only have the EHR data which needs to be copied.
Please remove this extra section. Lines 44 - 50